ucm2/HDA-Intel/Hdmi.conf

```diff
@@ -0,0 +1,79 @@
+# Use case Configuration for HDA-Intel
+
+If.hdmi1 {
+	Condition {
+		Type ControlExists
+		Control "iface=CARD,name='HDMI/DP,pcm=3 Jack'"
+	}
+	True {
+		SectionDevice."HDMI1" {
+			Comment "HDMI1/DP1 Output"
+
+			EnableSequence [
+				cset "name='IEC958 Playback Switch' on"
+			]
+
+			DisableSequence [
+				cset "name='IEC958 Playback Switch' off"
+			]
+
+			Value {
+				PlaybackPriority 500
+				PlaybackPCM "hw:${CardId},3"
+				JackControl "HDMI/DP,pcm=3 Jack"
+			}
+		}
+	}
+}
+
+If.hdmi2 {
+	Condition {
+		Type ControlExists
+		Control "iface=CARD,name='HDMI/DP,pcm=4 Jack'"
+	}
+	True {
+		SectionDevice."HDMI2" {
+			Comment "HDMI2/DP2 Output"
+
+			EnableSequence [
+				cset "name='IEC958 Playback Switch',index=1 on"
+			]
+
+			DisableSequence [
+				cset "name='IEC958 Playback Switch',index=1 off"
+			]
+
+			Value {
+				PlaybackPriority 600
+				PlaybackPCM "hw:${CardId},4"
+				JackControl "HDMI/DP,pcm=4 Jack"
+			}
+		}
+	}
+}
+
+If.hdmi3 {
+	Condition {
+		Type ControlExists
+		Control "iface=CARD,name='HDMI/DP,pcm=5 Jack'"
+	}
+	True {
+		SectionDevice."HDMI3" {
+			Comment "HDMI3/DP3 Output"
+
+			EnableSequence [
+				cset "name='IEC958 Playback Switch',index=2 on"
+			]
+
+			DisableSequence [
+				cset "name='IEC958 Playback Switch',index=2 off"
+			]
+
+			Value {
+				PlaybackPriority 700
+				PlaybackPCM "hw:${CardId},5"
+				JackControl "HDMI/DP,pcm=5 Jack"
```

Comment:
And change 5 to 8:

			        PlaybackPCM "hw:${CardId},8"
				JackControl "HDMI/DP,pcm=8 Jack"